publication-rest/src/test/java/no/unit/nva/publication/update/UpdatePublicationHandlerTest.java

```diff
@@ -378,13 +402,13 @@ void handlerThrowsExceptionWhenInputIsValidUserHasRightToEditAnyResourceInOwnIns
     }
 
     @Test
-    void handlerThrowsNotFoundWhenExternalClientTriesToUpdateResourcesCreatedByOthers()
-        throws IOException, BadRequestException {
+    void handlerReturnsForbiddenWhenExternalClientTriesToUpdateResourcesCreatedByOthers()
```

Comment:
I Think it is great to return FORBIDDEN for all users without update rights, instead of NOT_FOUND